### PR TITLE
docs: Adding codelab for Spanner with DocLoader, VectorStore  & Memory

### PR DESCRIPTION
**Pull request by gauravpurohit06**
Adding codelab for Spanner with DocLoader, VectorStore  & Memory

### CODE REVIEW

File: samples/langchain_quick_start.ipynb [hunk 1]
```diff
@@ -20,14 +22,1338 @@
...
+      "page_content='Wild Dog Ahishor Solomon Nagarjuna Akkineni, Dia Mirza, Saiyami Kher, Atul Kulakarni, Bilal Hussain, Ali Reza, Mayank Parakh A brash but brilliant Indian intelligence agent leads a covert operation to nab the mastermind behind a series of attacks threatening national security.' metadata={'show_id': 's1001', 'type': 'Movie', 'country': '', 'date_added': 'April 22, 2021', 'release_year': 2020, 'rating': 'TV-MA', 'duration': '126 min', 'listed_in': 'Action & Adventure, International Movies'}\n",
+      "page_content='Oloibiri Curtis Graham Olu Jacobs, Richard Mofe-Damijo, William R. Moses, Taiwo Ajai-Lycett, Ifeanyi Williams, Ivie Okujaye, Dayton Sinkia, Bradley Gordon After drilling depletes a small village, a corporation finds more oil in the region, igniting tensions among the community.' metadata={'show_id': 's1002', 'type': 'Movie', 'country': 'Canada, Nigeria, United States', 'date_added': 'April 21, 2021', 'release_year': 2015, 'rating': 'TV-14', 'duration': '86 min', 'listed_in': 'Dramas, International Movies, Thrillers'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "documents = await collect_async_items(loader.alazy_load())\n",
```
Comment (averikitsch) [inline samples/langchain_quick_start.ipynb:742]: Can we call `aload()` or `load()` so we don't have to define an additional method?

File: samples/langchain_quick_start.ipynb [hunk 2]
```diff
@@ -20,14 +22,1338 @@
...
+    "    with database.batch() as batch:\n",
+    "        batch_rows = rows[i : i + batch_size]\n",
+    "        batch.insert(\n",
+    "            table=table_name,\n",
+    "            columns=[\n",
+    "                \"show_id\",\n",
+    "                \"type\",\n",
+    "                \"title\",\n",
+    "                \"director\",\n",
+    "                \"casta\",\n",
```
Comment (averikitsch) [inline samples/langchain_quick_start.ipynb:631]: typo?

File: samples/langchain_quick_start.ipynb [hunk 3]
```diff
@@ -20,14 +22,1338 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 66,
+   "metadata": {
+    "id": "FEe9El7QMjHi"
+   },
+   "outputs": [],
+   "source": [
+    "vector_store_rows = read_csv_from_gcs(\n",
+    "    \"gaurav-mlops-exp\", \"netflix_titles_vector_embeddings.csv\"\n",
```
Comment (averikitsch) [inline samples/langchain_quick_start.ipynb:917]: bucket should be "cloud-samples-data"

File: samples/langchain_quick_start.ipynb (hunk 2 above)
Comment (gauravpurohit06) [reply samples/langchain_quick_start.ipynb:631]: cast is reserved keyword, updating it to `cast_list`

File: samples/langchain_quick_start.ipynb [hunk 4]
```diff
@@ -20,14 +22,1338 @@
...
+      "langchain-google-spanner 0.1.0 requires langchain-core<1.0.0,>=0.1.25, but you have langchain-core 0.1.23 which is incompatible.\u001B[0m\u001B[31m\n",
+      "\u001B[0mSuccessfully installed langchain-community-0.0.20 langchain-core-0.1.23 langsmith-0.0.87\n",
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# TODO: Update this to install from Pypi\n",
+    "%pip install git+https://github.com/googleapis/langchain-google-spanner-python.git\n",
+    "# %pip install langchain_google_spanner\n",
```
[the remark this answers is not in the document]
Comment (gauravpurohit06) [reply samples/langchain_quick_start.ipynb:275]: Actually not, using a static function which is not part of release.

File: samples/langchain_quick_start.ipynb [hunk 5]
```diff
@@ -20,14 +22,1094 @@
...
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {
+    "id": "CTks8Cy--93B"
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['83118fcc-6c38-4b93-9dab-03270c9cf2a9',\n",
```
Comment (tabbyl21) [inline samples/langchain_quick_start.ipynb:630]: I think that the output is still showing in this file, could you clear the cells first

**Review by tabbyl21 — APPROVED** [review]

**Review by averikitsch — APPROVED** [review]

**Pull request merged**